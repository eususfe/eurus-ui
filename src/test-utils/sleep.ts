/*
 * @Author: jiangxd
 * @Date: 2022-09-23 14:06:55
 * @LastEditTime: 2022-09-23 14:07:04
 * @LastEditors: jiangxd
 * @Description:
 * @FilePath: /eurus-ui/src/test-utils/sleep.ts
 */
const sleep = (time = 0) => new Promise(resolve => setTimeout(resolve, time));

export default sleep;
