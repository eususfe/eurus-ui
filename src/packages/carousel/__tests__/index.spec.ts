import { mount } from '@vue/test-utils';
import ECarousel from '..';

describe('carousel', () => {
  it('carousel snapshot', () => {
    const wrapper = mount(ECarousel);
    expect(wrapper.html()).toMatchSnapshot();
    wrapper.unmount();
  });
});
