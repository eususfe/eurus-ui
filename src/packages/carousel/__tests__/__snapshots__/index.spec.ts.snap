// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`carousel > carousel snapshot 1`] = `
"<div class=\\"eu-carousel\\" style=\\"height: 300px;\\">
  <transition-stub name=\\"carousel-arrow-left\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\"><button class=\\"eu-carousel-arrow eu-carousel-arrow--left\\"><span class=\\"eu-icon\\" role=\\"link\\" tabindex=\\"0\\" style=\\"font-size: 30px; color: #fff;\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" xmlns:xlink=\\"http://www.w3.org/1999/xlink\\" width=\\"1em\\" height=\\"1em\\" viewBox=\\"0 0 32 32\\"><path fill=\\"currentColor\\" d=\\"M10 16L20 6l1.4 1.4l-8.6 8.6l8.6 8.6L20 26z\\"></path></svg></span></button></transition-stub>
  <transition-stub name=\\"carousel-arrow-right\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\"><button class=\\"eu-carousel-arrow eu-carousel-arrow--right\\"><span class=\\"eu-icon\\" role=\\"link\\" tabindex=\\"0\\" style=\\"font-size: 30px; color: #fff;\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" xmlns:xlink=\\"http://www.w3.org/1999/xlink\\" width=\\"1em\\" height=\\"1em\\" viewBox=\\"0 0 32 32\\"><path fill=\\"currentColor\\" d=\\"M22 16L12 26l-1.4-1.4l8.6-8.6l-8.6-8.6L12 6z\\"></path></svg></span></button></transition-stub>
  <!---->
  <div role=\\"list\\" class=\\"eu-carousel__indicator eu-carousel__indicator--horizontal\\"></div>
</div>"
`;
